Sources/Sentry/PrivateSentrySDKOnly.mm

```diff
@@ -304,24 +305,43 @@ + (SentryBreadcrumb *)breadcrumbWithDictionary:(NSDictionary *)dictionary
     return [[SentryBreadcrumb alloc] initWithDictionary:dictionary];
 }
 
+#if SENTRY_HAS_UIKIT && !TARGET_OS_VISION
++ (SentrySessionReplayIntegration *)getReplayIntegration
```

Comment:
`m`: The return value of this method should be nullable as there is no guarantee that we always find an `SentrySessionReplayIntegration`.

Reply:
```suggestion
+ (nullable SentrySessionReplayIntegration *)getReplayIntegration
```